src/test/java/weatherApiTests/AuthKeyTest.java

```diff
@@ -1,46 +1,70 @@
 package weatherApiTests;
 
-import Utils.EndPoints;
-import org.testng.annotations.DataProvider;
-import org.testng.annotations.Test;
+import io.qameta.allure.*;
+import utils.EndPoints;
+import io.restassured.response.Response;
+import org.testng.annotations.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import static io.restassured.RestAssured.given;
-import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.*;
+
+@Epic("API tests for testWeather project")
+@Feature("Authentication verification")
 
 public class AuthKeyTest {
-    private EndPoints ed = new EndPoints();
+    private EndPoints endPoint1 = new EndPoints();
 
-    @DataProvider
-    public Object[][] requestData() {
-        return new Object[][]{
-                {null, 401},
-                {"4t5gdg534df453", 401},
-                {"0b4a9150dfd5e34fc0d8afa2a8d1629f", 401},
-                {"0b4a9150ecd5efe7c0d8afa2a8d1629e", 200}
+    private static final Logger LOG = (ch.qos.logback.classic.Logger) LoggerFactory.getLogger("com.testWeather.AuthKeyTest");
+    private static int expectedStatusCodeUnath = 401;
```

Comment:
> expectedStatusCodeUn**ath**
a tiny typo expectedStatusCodeUna**u**th

Reply:
Corrected in 2cee23d501c7b510812544be29eae12718646349